02-electric/electric-section-four/electric-section-four.css

```diff
@@ -1,32 +1,347 @@
-.electric-section-four-container{
-    display: grid;
-    grid-template-columns: 1fr;
-    justify-content: center;
-    row-gap: 15px;
-    padding: 25px
+.electric-section-four-container,
+.electric-section-four-subcontainer,
+.electric-section-four-title,
+.electric-section-four-part-one-container,
+.electric-section-four-part-one-image-container,
+.electric-section-four-part-one-copy-container,
+.electric-section-four-part-one-subtitle,
+.electric-section-four-part-one-paragraph-container,
+.electric-section-four-part-one-paragraph,
+.electric-section-four-part-two-container,
+.electric-section-four-part-two-image-container,
+.electric-section-four-part-two-copy-container,
+.electric-section-four-part-two-subtitle,
+.electric-section-four-part-two-paragraph-container,
+.electric-section-four-part-two-paragraph,
+.electric-section-four-part-three-container,
+.electric-section-four-part-three-image-container,
+.electric-section-four-part-three-copy-container,
+.electric-section-four-part-three-subtitle,
+.electric-section-four-part-three-paragraph-container,
+.electric-section-four-part-three-paragraph{
+    border:var(--temporary-border)
+}
+
+.electric-section-four-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--container-padding-mobile);
+  
+}
+
+@media (min-width:500px) {
+  .electric-section-four-container {
+    padding: var(--container-padding-desktop);
+  }
+}
+
+.electric-section-four-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+  "electric-section-four-title"
+  "electric-section-four-part-one-container"
+  "electric-section-four-part-two-container"
+  "electric-section-four-part-three-container";
+  justify-content: center;
+  row-gap: var(--subsection-container-row-gap-mobile);
+  column-gap:none;
+  padding: var(--subcontainer-padding-mobile);
+  border: solid 1px black;
+  background-color: var(--level1-background-color);
+}
+
+@media (min-width:500px){
+  .electric-section-four-subcontainer{
+    grid-template-columns: repeat(3,1fr);
+    grid-template-rows: auto 40px auto auto auto auto;
```

Comment:
this is the correct solution, well done. 
but we need to make sure the top of the part-two container is not touching the title (once the title is in the title container div) and that the row-gap is applying. which at the moment it doesnt seem to be.